source/files/stylesheets/adjustments.css

```diff
@@ -487,18 +343,12 @@ html:not(.csscalc) .layout-footer .footer-row-1 > .fg-col:last-child {
   height: auto;
 }
 
-.layout-footer .footer-row-1 .social-icon {
-  float: left;
```

Comment:
Hmm. Do you think the changed spacing looks bad, or should we just go for it? I can't actually discern the difference on Gecko or Blink, tbh.